qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -180,6 +180,7 @@ def addEdgeItem(self, edge, nodes, edges, highlight_level, same_label_siblings=F
         if 'style' in edge.attr:
             if edge.attr['style'] == 'invis':
                 return
+        style = edge.attr.get('style', 'solid')
```

Comment:
Since yoo choose the default also in the `EdgeItem` this line should not define a default but `None` instead.